### PR TITLE
Update to modern standards

### PR DESCRIPTION
**Pull request by terodox**
- Update to use babel@7
- Update to jest from jasmine/karma
- Update dependencies to latest versions

* [x] Change is tested locally
* [x] [WIP] flag is removed from the title

### CODE REVIEW

**Review by mattquinlan440 — DISMISSED** [review]
Thanks for tackling this!

File: PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -1,3 +1,2 @@
 * [ ] Change is tested locally
-* [ ] Demo is updated to exercise change (if applicable)
```
Comment (jtheriault) [inline PULL_REQUEST_TEMPLATE.md]: Can we keep this and introduce a demo later?

File: PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -1,3 +1,2 @@
 * [ ] Change is tested locally
```
Comment (terodox) [inline PULL_REQUEST_TEMPLATE.md:1]:
```suggestion
* [ ] Change is tested locally
* [ ] Demo is updated to exercise change (if applicable)
```

File: PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (terodox) [reply PULL_REQUEST_TEMPLATE.md]: I'm fine with that.

Addressed in 223580e

**Review by jtheriault — APPROVED** [review]
Thank you for doing this

**Pull request merged**